opentelemetry-ext-google-cloud/README.rst

```diff
@@ -0,0 +1,87 @@
+OpenTelemetry Google Cloud Integration
+======================================
+
+.. image:: https://badge.fury.io/py/opentelemetry-ext-google-cloud.svg
+    :target: https://badge.fury.io/py/opentelemetry-ext-google-cloud
+
+This library provides support for:
+
+- Exporting traces to Google Cloud Trace
+- Exporting metrics to Google Cloud Monitoring
+- Detecting GCP resources like GCE, GKE, etc.
+- Propagating trace context in the legacy Google Cloud format
+
+Installation
+------------
+
+::
+
+    pip install opentelemetry-ext-google-cloud
+
+Usage
+-----
+
+Traces
+
+.. code:: python
+
+    from opentelemetry import trace
```

Comment:
Should use a `literalinclude` to `basic_trace` here instead.

Reply:
I don't think it will work since this RST file isn't processed by sphinx. I think it's just included in the wheel you send to pypi

Reply:
Ya, won't even let me upload to testpypi: `400 The description failed to render for 'text/x-rst'.`